get-access-token.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+tokenUrl=$(cat /usr/oidc/credentials.json | jq -r '.tokenUrl')
+if [ -z "${tokenUrl}" ]; then
+  exit 0
+fi
+
+clientId=$(cat /usr/oidc/credentials.json | jq -r '.clientId')
+clientSecret=$(cat /usr/oidc/credentials.json | jq -r '.clientSecret')
+
+response=$(curl --user ${clientId}:${clientSecret} -d "grant_type=client_credentials" ${tokenUrl})
```

Comment:
We might need a configurable `-k` here, otherwise curl rejects the ingress controller cert in the local env:

```
+ ./get-access-token.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```